.cargo/config

```diff
@@ -1,11 +1,3 @@
-[alias]
```

Comment:
I really wish we could leave this in because it works and it's what I use. This may sound petty but I'm also coming at this as someone who was fairly new to libtock-rs. I had problems getting stared and some of that was not understanding cargo but more of it was the fact that we didn't have an end-to-end way to build everything cleanly. 'PLATFORM=blah cargo rrv32imac hello_world' now works fine. 

I actually think introducing people to cargo is important simply because this is a rust project. cargo and build.rs are fairly standard rust constructs and are well documented (https://doc.rust-lang.org/cargo/reference/index.html). The nice thing about build.rs is that we can take advantage of rust's features to catch errors. I've wasted many hours debugging both make and shell script errors that would have been caught with a stricter language. We should focus on making  our rust/cargo easy for new contributors to use rather than assuming make is the only option. If I, coming from a make background, can learn how cargo works I think other contributors can too. To be clear, this isn't "I had it hard so everyone else should too" but that once I learned a little how cargo works what .cargo/config and build.rs were doing actually made a lot of sense. If we really want a Makefile to make things eaiser, I'd like it to be as thin as possible and optimize for using cargo over make which would mean keeping our aliases around.

Reply:
I have left the aliases in so you can keep using them.

Reply:
Just so it's clear this PR doesn't change cargo, you can still use all the same commands and the make infrastrucutre will still call all of the same commands. Make will also tell you what it is calling so you can see and manually run the commands. Make is just a wrapper for the large number of different possible cargo options. This way it's easier for new people to start and easier for developers to not have to remember all the different possible combinations and which ones work/don't work.